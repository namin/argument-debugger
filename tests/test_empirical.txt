Crime rates have increased by 20% in our city.
Therefore, we need to hire more police officers.

All bachelors are unmarried men.
John is a bachelor.
Therefore, John is an unmarried man.

Video games cause violence in children.
My nephew plays violent games.
Therefore, my nephew will become violent.