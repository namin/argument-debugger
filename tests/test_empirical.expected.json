{
  "description": "Tests empirical claim detection - crime and video games need evidence, bachelor definition doesn't",
  "must_find": ["missing_link", "unsupported_premise"],
  "may_find": [],
  "must_not_find": ["circular", "false_dichotomy"]
}